<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestClientClass</class>
 <widget class="QWidget" name="TestClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestClient</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutMain">
   <item>
    <layout class="QHBoxLayout" name="hLayoutConnect">
     <item>
      <widget class="QLabel" name="lblIp">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtIP">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spPort">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>3240</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gboxTc">
     <property name="title">
      <string>TC</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="edtMsg"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnGetPos">
          <property name="text">
           <string>GetPos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetPos">
          <property name="text">
           <string>SetPos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTest">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTest2">
          <property name="text">
           <string>Test2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblSay">
          <property name="text">
           <string>Say:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtSay"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testclient.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot_GetPos()</slot>
  <slot>slot_SetPos()</slot>
  <slot>slot_Test()</slot>
  <slot>slot_Test2()</slot>
  <slot>slot_Disconnect()</slot>
 </slots>
</ui>
